string name
uint8[] data
---
bool accepted